content/en/docs/concepts/policy/resource-quotas.md

```diff
@@ -610,17 +610,29 @@ plugins:
         values: ["cluster-services"]
 ```
 
-Now, "cluster-services" pods will be allowed in only those namespaces where a quota object with a matching `scopeSelector` is present.
-For example:
+Then, create a corresponding resource quota object in `kube-system` namespace:
 
-```yaml
+```shell
+$ cat ./quota.yml
```

Comment:
```suggestion
$ cat ./quota.yml
```
Distinguish between 'command'and 'output'.

Reply:
Done.